rgw/v2/tests/s3_swift/test_encrypted_bucket_chown.py

```diff
@@ -0,0 +1,205 @@
+"""
+
+test chown for a bucket encrypted with sse-s3
+Usage: test_encrypted_bucket_chown.py -c <input_yaml>
+<input_yaml>
+    test_encrypted_bucket_chown.yaml
+
+Operation:
+    Create 2 users and create a bucket with user1 credentials
+    enable per-bucket sse-s3 encryption with vault backend
+    change the ownership of the bucket to user2 and then upload objects
+    test objects uploaded are encrypted with AES256. 
+
+"""
+
+import os
+import sys
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+import argparse
+import hashlib
+import json
+import logging
+import random
+import time
+import traceback
+import uuid
+
+import v2.lib.manage_data as manage_data
+import v2.lib.resource_op as s3lib
+import v2.utils.utils as utils
+from v2.lib.exceptions import RGWBaseException, TestExecError
+from v2.lib.resource_op import Config
+from v2.lib.rgw_config_opts import CephConfOp, ConfigOpts
+from v2.lib.s3.auth import Auth
+from v2.lib.s3.write_io_info import BasicIOInfoStructure, BucketIoInfo, IOInfoInitialize
+from v2.tests.s3_swift import reusable
+from v2.tests.s3_swift.reusables import server_side_encryption_s3 as sse_s3
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+from v2.utils.utils import RGWService
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def test_exec(config, ssh_con):
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    write_bucket_io_info = BucketIoInfo()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    ceph_conf = CephConfOp(ssh_con)
+    rgw_service = RGWService()
+
+    # create user
+    if config.user_count is 2:
```

Comment:
@viduship, please include else condition, which errors out incase there are more than 2 users in user_count from conf file.

Reply:
done